lib/perl/Genome/Model/SingleSampleGenotype/Command/HaplotypeCaller/BucketIterator.pm

```diff
@@ -56,6 +56,11 @@ sub _run {
             $self->fatal_message('Could not %s for interval %s.', $mode, $interval);
         }
 
+        unless($ENV{UR_DBI_NO_COMMIT}) {
+            #save our progress after creating a new result
```

Comment:
Is it necessary to check if no commit is set? Doesn't that happen under the hood in UR?